org.ghidra_sre.Ghidra.json

```diff
@@ -14,9 +14,11 @@
         "--socket=x11",
         "--socket=pulseaudio",
         "--device=dri",
+        "--allow=devel",
         "--filesystem=home",
         "--env=PATH=/usr/bin:/app/bin:/app/jdk/bin",
-        "--env=JAVA_HOME=/app/jdk"
+        "--env=JAVA_HOME=/app/jdk",
+        "--env=LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu"
```

Comment:
Why is this needed?

Reply:
Otherwise, Java is unable to find `libutil.so`

Reply:
`/usr/lib/x86_64-linux-gnu` should already be in the default library path (through ld.so.conf). Isn't there a Java specific option to use instead?

Reply:
I don't know

Reply:
Using `LD_CONFIG=/etc/ld.so.conf` doesn't seem to work.

Reply:
Why you don't like using `LD_LIBRARY_PATH`?